base_app/base.py

```diff
@@ -36,10 +39,25 @@ def get_random():
     params = format_params(request.args)
     return service.get_recommendation(params )
 
+@app.route('/api/v1/notify', methods=['GET'])
```

Comment:
Awesome! I want to make sure I'm understanding this, so let me know if this isn't exactly how it works: So we assign this route to the "Send to Friends" button, FE sends params as 'from', 'to', and 'body', a text is created, and then gets sent. Is 'body' just the restaurant name?

Reply:
body is going to be "Brothers Bar at 1900 Market St. Denver, CO"

Reply:
so (restaurant.name) at (restaurant.location)